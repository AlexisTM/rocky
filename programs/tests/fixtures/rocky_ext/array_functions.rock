Let the roses at 0 be "red"
Let the roses at 1 be "purple"
Let the roses at 2 be "white"
While the roses alike my rose
Say "A " with my rose with " rose for my darling"
(end while)

(sorting)
Let a bouquet be the roses sorted
While a bouquet alike a flower
Say "A " with a flower with " bouquet"
(end while)

Let the bouquet be sorted the roses
While the bouquet alike a flower
Say "A " with a flower with " bouquet"
(end while)

Say count of the roses
Say length of the roses
Say height of the roses

Say last of the roses

While the roses alike my rose
Say "A " with my rose with " rose for my darling"
(end while)
