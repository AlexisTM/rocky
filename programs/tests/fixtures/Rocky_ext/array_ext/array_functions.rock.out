A red rose for my darling
A purple rose for my darling
A white rose for my darling
A purple bouquet
A red bouquet
A white bouquet
A purple bouquet
A red bouquet
A white bouquet
3
3
3
white
A red rose for my darling
A purple rose for my darling
A white rose for my darling
